src/integrations/admin/indexing-tool-integration.php

```diff
@@ -145,6 +144,21 @@ public function enqueue_scripts() {
 		$this->asset_manager->localize_script( 'indexation', 'yoastIndexingData', $data );
 	}
 
+	/**
+	 * The error to show if optimization failed.
+	 *
+	 * @return string The error to show if optimization failed.
+	 */
+	protected function render_indexing_error() {
+		$presenter = new Indexing_Error_Presenter(
```

Comment:
Can't we inject the Indexing_Error_Presenter using dependency injection so we don't have to instantiate it manually?

Reply:
Unfortunately not. [The presenters (e.g. all the classes from the `src/presenters` folder) are excluded from the dependency injection container](https://github.com/Yoast/wordpress-seo/blob/trunk/config/dependency-injection/services.php#L56).

I'm not 100% sure what the reason behind that is, though.